debezium-server-iceberg-sink/src/main/java/io/debezium/server/iceberg/IcebergConfig.java

```diff
@@ -1,26 +1,38 @@
 package io.debezium.server.iceberg;
 
+import com.google.common.collect.ImmutableList;
 import io.quarkus.runtime.annotations.ConfigRoot;
 import io.smallrye.config.ConfigMapping;
 import io.smallrye.config.WithDefault;
 import io.smallrye.config.WithName;
+import org.apache.commons.lang.StringUtils;
 import org.apache.iceberg.CatalogProperties;
 
+import java.util.Arrays;
 import java.util.List;
 import java.util.Map;
 import java.util.Optional;
 
+import static java.util.stream.Collectors.toList;
 import static org.apache.iceberg.TableProperties.DEFAULT_FILE_FORMAT;
 import static org.apache.iceberg.TableProperties.DEFAULT_FILE_FORMAT_DEFAULT;
 
 @ConfigRoot
 @ConfigMapping
 public interface IcebergConfig {
   String PROP_PREFIX = "debezium.sink.iceberg";
+  String TABLE_PROP_PREFIX = "debezium.sink.iceberg.table";
+  String COMMA_NO_PARENS_REGEX = ",(?![^()]*+\\))";
 
   @WithName(PROP_PREFIX)
   Map<String, String> icebergConfigs();
 
+  @WithName(TABLE_PROP_PREFIX)
+  Map<String, IcebergTableConfig> tableConfigs();
```

Comment:
this could be optional `Optional<Map<String, IcebergTableConfig>>`

Reply:
> with that additional test is needed to ensure this comma is properly handled `bucket(16\, user_id),bucket(16\, country_id)`

sorry this was for the filed in the IcebergTableConfig class. the Quarkus config can read and separate given comma separated  partition settings 

`Optional<List> partitionBy();`